README.md

```diff
@@ -1,454 +1,35 @@
-# NVIDIA GPU feature discovery
+# GPU Feature Discovery
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/NVIDIA/gpu-feature-discovery)](https://goreportcard.com/report/github.com/NVIDIA/gpu-feature-discovery)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+**MIGRATION Notice**
 
-## Table of Contents
+This repository contains the source code and documentation for GPU Feature Discovery. As of [26 Feb 2024](https://github.com/NVIDIA/k8s-device-plugin/releases/tag/v0.15.0), this project has been **archived**. Development will now continue at [k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin).
 
-- [Overview](#overview)
-- [Beta Version](#beta-version)
-- [Prerequisites](#prerequisites)
-- [Quick Start](#quick-start)
-  * [Node Feature Discovery (NFD)](#node-feature-discovery-nfd)
-  * [Preparing your GPU Nodes](#preparing-your-gpu-nodes)
-  * [Deploy NVIDIA GPU Feature Discovery (GFD)](#deploy-nvidia-gpu-feature-discovery-gfd)
-    + [Daemonset](#daemonset)
-    + [Job](#job)
-  * [Verifying Everything Works](#verifying-everything-works)
-- [The GFD Command line interface](#the-gfd-command-line-interface)
-- [Generated Labels](#generated-labels)
-  * [MIG 'single' strategy](#mig-single-strategy)
-  * [MIG 'mixed' strategy](#mig-mixed-strategy)
-- [Deployment via `helm`](#deployment-via-helm)
-    + [Installing via `helm install`](#installing-via-helm-install)
-      - [Deploying Standalone](#deploying-standalone)
-      - [Deploying as a subchart of the NVIDIA device plugin](#deploying-as-a-subchart-of-the-nvidia-device-plugin)
-    + [Deploying via `helm install` with a direct URL to the `helm` package](#deploying-via-helm-install-with-a-direct-url-to-the-helm-package)
-- [Building and running locally with Docker](#building-and-running-locally-with-docker)
-- [Building and running locally on your native machine](#building-and-running-locally-on-your-native-machine)
+## Archived Status
 
-## Overview
+This project is no longer actively maintained. While the codebase remains available for reference purposes, no further updates or contributions are expected.
```

Comment:
This is not true. GFD as a project is still maintained, it's just not developed in this repo.

Reply:
rewritten